README.md

```diff
@@ -217,5 +217,7 @@ Preview builds (aka 'nightly' builds) are distributed using https://feedz.io/ fo
 
 ## Release process
 
-Before making a new release the version in the [csproj](Consul/Consul.csproj), in the [README](README.md), in the [CI pipeline](.github/workflows/ci.yml) and in the [CHANGELOG](CHANGELOG.md) files should be updated.
-The new versions can be released using the [releases](https://github.com/G-Research/consuldotnet/releases) page.
+1. Before making a new release, make sure that the version in [Directory.Build.props](Directory.Build.props) is up to date.
+2. Then, draft a new [GitHub Release](https://github.com/G-Research/consuldotnet/releases) with a list of changes contained in the new version. This list of changes should be committed to the [CHANGELOG](CHANGELOG.md) file for posterity.
```

Comment:
We should probably make it explicit that 'Unreleased' gets renamed to the version number in the changelog, so that when reading the changelog for a given tag, it looks neat.

Reply:
I tried something, let me know if it's clearer.